packages/app/src/systems/Home/components/UserAddressCard/UserAddressCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { cssObj } from '@fuel-ui/css';
+import { Avatar, Button, Card, Copyable } from '@fuel-ui/react';
+import { useCallback } from 'react';
+
+type Props = {
+  account: string;
+};
+
+export function UserAddressCard({ account }: Props) {
+  const copyAccount = useCallback(() => {
+    navigator.clipboard.writeText(account as string);
+  }, []);
+
+  return (
+    <Card css={styles.wrapper}>
+      <Avatar.Generated hash={account as string} size="2xl" background="fuel" />
```

Comment:
`account` is a `string` already. do we need to `as string` ?

Reply:
change prop name to `address`.

as we use `account` in other places refferring to `Account` type, it can be confusing.